src/components/hero/_macro.njk

```diff
@@ -99,7 +96,13 @@
             <svg class="hero__circle-lined" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 243 243" focusable="false" aria-hidden="true">
```

Comment:
SVG still needs removing here and isnt currently showing image in example

![image](https://user-images.githubusercontent.com/42928680/109821798-400f0c80-7c2e-11eb-8bc6-d193a7e72dba.png)

Reply:
Let me take a look at this again.

Reply:
In regards the image, I think this is a CDN issue. The same approach has been used for the other examples.